qmulticast/protocols/multipartiteprotocol.py

```diff
@@ -29,9 +30,14 @@ class MultipartiteProtocol(NodeProtocol):
     If node has recived its required qubit, return a (classical) flag to source to say its happy
     """
 
-    def __init__(self, node: Node, name: Optional[str] = None, source: bool = False, routing_table = None):
-        """Initialise the protocol wiht information about the node.
-
+    def __init__(
+        self,
+        node: Node,
+        name: Optional[str] = None,
+        source: bool = False,
+        receiver: bool = True,
+        routing_table = None):
```

Comment:
This could be assigned to a property of the node. Are we ever going to vary the routing table between protocol instances?